### ISSUE TITLE
Missing accessibility options in EnrichedMarkdownTextInput

### ISSUE THREAD

**Issue by mmiedlarz** [software-mansion-labs/react-native-enriched-markdown#521]
**Describe the bug**
There are two issues that are related to a11y while using TextInput:
1. Missing any RN accessibility props to pass to EnrichedMarkdownTextInput
2. While in VoiceOver mode, it lacks "Text Input" role, so it's not announced as editable by VoiceOver

**To Reproduce**
1. Render two components: EnrichedMarkdownTextInput and regular RN TextInput on single screen.
2. Enable VoiceOver (on iOS - by pressing side button 3 times). It might be helpful to hide curtain so you'll see what's on the screen.
3. Using swipe-left and swipe-right gesture, select EnrichedMarkdownTextInput - notice that you're able to select it but VoiceOver will miss text input information.
4. Select regular TextInput - notice that it'll announce correct type

**Expected behavior**
1. Ability to provide accessibility props to EnrichedMarkdownTextInput
2. Correct VoiceOver type

**Screenshots**

**Device (please complete the following information):**
 - Any iOS device

**Additional context**
Attaching video of the issue below:
https://github.com/user-attachments/assets/ef8eda9d-7f13-4f44-b541-a5b80da8ec0e

**Comment by eszlamczyk** [comment]
Hi @mmiedlarz thanks for the bug report! This should be fixed in next nightly release!

**Comment by eszlamczyk** [comment]
Readability done in #526

Note: improving the "writeability" on IOS will be much harder to do and would require doing a bunch of changes to architecture of input, so we will not pursue it for now
